Cargo.toml

```diff
@@ -9,3 +9,12 @@ members = [
     "piet-web",
     "piet-web/examples/basic"
 ]
+
+default-members = [
+    "piet",
+    "piet-cairo",
+    "piet-common",
+    "piet-test",
+    "piet-web",
+    "piet-web/examples/basic"
+]
```

Comment:
What's the significance of this change?

Reply:
Yes, that's where it came from.

Reply:
Perhaps it also makes sense to remove piet-cairo from this list and have users explicitly choose a backend with --features.